src/TableContent.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Table } from 'evergreen-ui'
+import URI from 'urijs'
+
+import tableHeaderCells from './tableHeaderCells'
+
+const TableContent = ({enableIndex, tableHeader, searchedColumnIndex, searchQuery, orderedColumnIndex,
+  ascendingOrder, enableDownload, checkedRows, currentPageData, host, entriesPerPage, currentPage,
+  tableHeaderOnClick, tableHeaderOnChange, downloadOnChange}) =>
+  <div className={`row expanded`}>
+    <div className={`small-12 columns`} >
+      <Table border>
+        <Table.Head>
+          {
+            enableIndex && <Table.TextHeaderCell key={`index`} flexBasis={100} flexShrink={100} flexGrow={100}>Index</Table.TextHeaderCell>
```

Comment:
I really think we could remove the `enableIndex` and the index column. It serves us very little purpose and I don’t think we’re ever going to use it.